Assets/Scripts/Player/TopDownMovement.cs

```diff
@@ -22,6 +22,7 @@ void Start()
 
     void Update()
     {
+        if (GameSettings.gameState != GameState.InGame){return;}
```

Comment:
Should stop all movement as well when the game isn't running.

Reply:
Fixed